tests/jax/components/executing/action_selection.py

```diff
@@ -0,0 +1,237 @@
+        self, agent: str, observation: NestedArray, state: NestedArray = None
+    ) -> NestedArray:
+        action_info = "action_info_" + str(agent) + "_" + str(observation)
+        policy_info = "policy_info_" + str(agent) + "_" + str(observation)
+        return action_info, policy_info
+
+
+@pytest.fixture
+def mock_executor() -> Executor:
+    """Mock executore component."""
+    return MockExecutor()
+
+
+# Test initiator
+def test_constructor(dummy_config: type) -> None:
+    """Test adding config as an attribute
+
+    Args:
+        dummy_config
+    """
+    ff_executor_select_action = FeedforwardExecutorSelectAction(config=dummy_config)
+    assert ff_executor_select_action.config.parm_0 == dummy_config.parm_0
+
+
+def test_constructor_empty_config(dummy_empty_config: type) -> None:
+    """Test adding empty config as an attribute
+
+    Args:
+        dummy_empty_config: ExecutorSelectActionConfig()
+    """
+    ff_executor_select_action = FeedforwardExecutorSelectAction(
+        config=dummy_empty_config
+    )
+    assert (
+        ff_executor_select_action.config.__dict__
+        == ExecutorSelectActionConfig().__dict__
+    )
+
+
+# Test on_execution_select_actions
+def test_on_execution_select_actions_with_empty_observations(
+    mock_empty_executor: Executor, dummy_config: type
+) -> None:
+    """Test on_execution_select_actions with empty observations
+
+    Args:
+        mock_empty_executor: executor with no observations and no agents
+    """
+    ff_executor_select_action = FeedforwardExecutorSelectAction(dummy_config)
+    ff_executor_select_action.on_execution_select_actions(executor=mock_empty_executor)
+
+    assert mock_empty_executor.store.actions_info == {}
+    assert mock_empty_executor.store.policies_info == {}
+
+
+def test_on_execution_select_actions(
+    mock_executor: Executor, dummy_config: type
+) -> None:
+    """Test on_execution_select_actions.
+
+    Args:
+        dummy_config: config
+        mock_executor: Executor
+    """
+
+    ff_executor_select_action = FeedforwardExecutorSelectAction(dummy_config)
+    ff_executor_select_action.on_execution_select_actions(executor=mock_executor)
+
+    assert (
+        mock_executor.store.actions_info["agent_0"]
+        == "action_info_agent_0_observation_0"
+    )
+    assert (
+        mock_executor.store.actions_info["agent_1"]
+        == "action_info_agent_1_observation_1"
+    )
+    assert (
+        mock_executor.store.actions_info["agent_2"]
+        == "action_info_agent_2_observation_2"
+    )
+
+    assert (
+        mock_executor.store.policies_info["agent_0"]
+        == "policy_info_agent_0_observation_0"
+    )
+    assert (
+        mock_executor.store.policies_info["agent_1"]
+        == "policy_info_agent_1_observation_1"
+    )
+    assert (
+        mock_executor.store.policies_info["agent_2"]
+        == "policy_info_agent_2_observation_2"
+    )
+
+
+def test_on_execution_select_actions_param(dummy_config: type) -> None:
```

Comment:
This test doesn't feel necessary to me. You're just asserting that it crashes when something without `store` and `select_action` is passed through.